packages/core/src/jobs/job-definitions/batchEvaluations/runEvaluationJob.ts

```diff
@@ -1,15 +1,51 @@
-import { env } from '@latitude-data/env'
 import { Job } from 'bullmq'
 
+import { RunErrorCodes } from '../../../constants'
+import { Result } from '../../../lib'
 import { queues } from '../../../queues'
 import {
   DocumentLogsRepository,
   EvaluationsRepository,
 } from '../../../repositories'
+import { isChainError } from '../../../services/chains/ChainStreamConsumer'
 import { runEvaluation } from '../../../services/evaluations/run'
 import { WebsocketClient } from '../../../websockets/workers'
 import { ProgressTracker } from '../../utils/progressTracker'
 
+function throwIfUnknownError(
+  error: unknown | undefined,
+): asserts error is Error {
+  const isAllGood =
+    !error || (isChainError(error) && error.errorCode !== RunErrorCodes.Unknown)
+
+  if (isAllGood) return
+
+  throw error
```

Comment:
This is important. We only throw an error in an evaluation run in 2 situations

1. Is a ChainError of type unknown. This is the catch in `runChain` not knowing what happened to this has to bubble to Sentry. Hopefully. 
2. is not a `ChainError` so it means something exploded.


Any other `ChainError` is known, saved and should be ignored by Sentry

Reply:
nice maybe leave this comment in the code actually, it's helpful

Reply:
Doc added